route/route.go

```diff
@@ -191,6 +199,45 @@ func (r *Route) PathSelection() {
 	r.updateEqualPathCount()
 }
 
+// Euql compares Are two routes and return true if they are equal
```

Comment:
Rephrase please :)

Reply:
done... Just realized how silly this comment was xD